xcube/core/store/fs/store.py

```diff
@@ -187,12 +189,25 @@ def get_data_store_params_schema(cls) -> JsonObjectSchema:
 
     @classmethod
     def get_data_types(cls) -> Tuple[str, ...]:
-        return tuple(set(_FILENAME_EXT_TO_DATA_TYPE_ALIAS.values()))
+        return tuple(set(
+            data_type
+            for types_tuple in _FORMAT_TO_DATA_TYPE_ALIASES.values()
+            for data_type in types_tuple
+        ))
 
     def get_data_types_for_data(self, data_id: str) -> Tuple[str, ...]:
         self._assert_valid_data_id(data_id)
-        data_type_alias, _, _ = self._guess_accessor_id_parts(data_id)
-        return data_type_alias,
+        data_type_alias, format_id, protocol = \
+            self._guess_accessor_id_parts(data_id)
```

Comment:
This doesn't look right to me. self._guess_accessor_id_parts could be able to return more than one data_type_alias, however, just the first one is returned. We are not necessarily returning all data types for data in this method.

Reply:
This is the logic that is currently implemented and it has been like this for a long time. Please post a report, if you found an issue and describe the problem that you observe.

Reply:
No, this is about a newly introduced change. See my new comment in _guess_accessor_id_parts.